.repo-metadata.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "aiplatform",
-  "name_pretty": "Cloud AI Platform",
-  "product_documentation": "https://cloud.google.com/ai-platform-unified/docs",
+  "name_pretty": "Vertex AI",
```

Comment:
Vertex AI looks like a combined API for AutoML and AI Platform. This library only consist of AI Platform and Auto ML has a separate library. Is there a plan to combine the libraries?